docs/release-notes.md

```diff
@@ -0,0 +1,41 @@
+**Trino Gateway documentation**
+
+<table>
+  <tr>
+    <td><a href="design.md">Design</a></td>
+    <td><a href="development.md">Development</a></b></td>
+    <td><a href="security.md">Security</a></td>
+    <td><a href="operation.md">Operation</a></td>
+    <td><a href="gateway-api.md">Gateway API</a></td>
+    <td><a href="resource-groups-api.md">Resource groups API</a></td>
+    <td><a href="routing-rules.md">Routing rules</a></td>
+    <td><a href="references.md">References</a></td>
+    <td><b><a href="release-notes.md">Release notes</a></b></td>
+  </tr>
+</table>
+
+# Release notes
+
+## Trino Gateway 3 (xx Sep 2023)
+
+The first release of Trino Gateway is based on the Presto Gateway 1.9.5 codebase
+[#4](https://github.com/trinodb/trino-gateway/pull/4), and a refactoring to
+Trino Gateway. In addition, the following updates are included:
+
+### New features
```

Comment:
Maybe we want this header .. but we could also just use the same implicit ordering as in the Trino release notes.

Reply:
I think having it as a part of the release notes definitely makes sense.

Reply:
LGTM :)